### PR TITLE
Subscribe functionality

### PR DESCRIPTION
**Pull request by velvelshteynberg**
Subscribers Functionality complete.

### CODE REVIEW

**Comment by velvelshteynberg** [comment]
Made a mistake

**Comment by velvelshteynberg** [comment]
Please review.

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "about_us#home"
+  get 'about_us/about_us', to: 'about_us#about_us'
+  get 'about_us/contact', to: 'about_us#contact_us'
+  resources :events, only: [:index, :new, :show]
+  resource :sessions, only: [:new, :delete]
+  resources :users, only: [:new]
+  resources :subscribers, only: [:create]
```
Comment (Shane0018) [inline config/routes.rb:9]: should there also be delete for subscribers who want to unsubscribe?

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "about_us#home"
+  get 'about_us/about_us', to: 'about_us#about_us'
+  get 'about_us/contact', to: 'about_us#contact_us'
+  resources :events, only: [:index, :new, :show]
```
Comment (Shane0018) [inline config/routes.rb]: should there also be edit?

File: config/routes.rb (hunk 1 above)
Comment (velvelshteynberg) [reply config/routes.rb:9]: No. I dont think so. We would have to authenticate the subscriber which adds an entirely new level of complication.

File: config/routes.rb (hunk 2 above)
Comment (velvelshteynberg) [reply config/routes.rb]: I added the edit route.

**Comment by velvelshteynberg** [comment]
Anything else?

**Comment by Shoshana01** [comment]
should there be an edit for users too?

**Review by Shane0018 — APPROVED** [review]

**Pull request merged**